api-controllers/ServiceFabrikApiController.js

```diff
@@ -594,6 +602,10 @@ class ServiceFabrikApiController extends FabrikBaseController {
         plan_id: planId,
         tenant_id: tenantId
       }))
+      .then(() => this.getBoshRestoreConfig(serviceId))
+      .then(disabled => {
```

Comment:
Hi @kamath-prasad @abh1kg , added a configMap check here before determining restore type for the service. A _service specific_ parameter named conforming to pattern `disable_bosh_based_restore_serviceName` in `sfconfig` configMap (as introduced in  https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/495) can be now used as a runtime hard reset switch to old restore mechanism for a given service. Will shortly raise corresponding PR on sfpc.